pkg/cli/admin/catalog/build.go

```diff
@@ -0,0 +1,57 @@
+package catalog
+
+import (
+	"github.com/operator-framework/operator-registry/pkg/appregistry"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	imgappend "github.com/openshift/oc/pkg/cli/image/append"
+)
+
+var (
+	buildLong = templates.LongDesc(`
+			Builds a catalog container image from a collection operator manifests.
+
+			Extracts the contents of a collection of operator manifests to disk, and builds them into
+			an operator registry catalog image.
+		`)
+)
+
+func NewBuildImage(streams genericclioptions.IOStreams) *cobra.Command {
```

Comment:
This command is not following the patterns set by all commands, this means **every** command needs to:
1. Have `<command-name>Options` struct with all the necessary flags used by it. Flags should be mapped 1-to-1 in this struct. Additional necessary fields are allowed. 
2. Have these methods:
  * `Complete` - responsible for completing the options struct from 1 (required)
  * `Verify` - responsible for performing validation (optional)
  * `Run` - responsible for actual execution
3. Each should use `kcmd.util.CheckErrr` when running ^
4. Use `Run` from cobra.
See other commands for examples.